### PR TITLE
added pvresize steps

### PR DESCRIPTION
**Pull request by g0rgamesh**

### CODE REVIEW

File: etc/kayobe/environments/ci-multinode/README.md [hunk 1]
```diff
@@ -28,12 +28,19 @@ Also storage nodes which back OpenStack such as Cinder, Glance and Nova are apar
     ```
     sudo rm -fr /home/stack
     ```
-    5. For the storage nodes ensure the hostname is not FQDN
+    5. If you are using Centos ensure pvresize has been executed on your partition (computes and controllers BMs)
```
Comment (markgoddard) [inline etc/kayobe/environments/ci-multinode/README.md:31]: Why not storage nodes?

Comment (markgoddard) [reply etc/kayobe/environments/ci-multinode/README.md:31]: Oh, I see it's a separate step.

Comment (markgoddard) [reply etc/kayobe/environments/ci-multinode/README.md:31]: This needs to be fixed in the playbook, since it affects every CentOS deployment.

**Review by markgoddard — CHANGES_REQUESTED** [review]

File: etc/kayobe/environments/ci-multinode/README.md (hunk 1 above)
Comment (g0rgamesh) [reply etc/kayobe/environments/ci-multinode/README.md:31]: Which playbook would that be?

Comment (markgoddard) [reply etc/kayobe/environments/ci-multinode/README.md:31]: etc/kayobe/ansible/growroot.yml

Greg had a change to always run the pvresize.

Comment (g0rgamesh) [reply etc/kayobe/environments/ci-multinode/README.md:31]: Seems like it still doesn't trigger

Comment (grzegorzkoper) [reply etc/kayobe/environments/ci-multinode/README.md:31]: I didn't commit it yet to any branch.
Just remove the last line in  ( conditional )
ansible/growroot.yml

Comment (grzegorzkoper) [reply etc/kayobe/environments/ci-multinode/README.md:31]: I'll push it to multinode-env in a sec


Edit:
https://github.com/stackhpc/stackhpc-kayobe-config/pull/203

**Pull request closed without merging**